cypress/fixtures/work-orders/filter.json

```diff
@@ -0,0 +1,197 @@
+{
```

Comment:
This file should already exists under `fixtures/filter/work-order.json` but happy if we want to move the file location to this instead. Also I think the formatting looks slightly off just at a glance.

Reply:
Ugh, I just saw that the request wasn't being intercepted and assumed that it didn't exist - will update to point to existing file.